### PR TITLE
ACN_CD_QualysVMFix4

### PR DESCRIPTION
**Pull request by chicduong**
Fixes #

## Proposed Changes

  - change the function.json in the ARM template zip package, timer to 60 mins 
  - modifications to the run.ps1 code, move newcustomObjects array outside the if statement. Remove Write-Output $json, unnecessary line.
- updated run.ps1 in the ARM template zip package

### CODE REVIEW

**Comment by chicduong** [comment]
@preetikr the function.json has been updated. The update needed to be made in the function.json that is within the ARM template zip, so it's not showing up as the individual file.

**Comment by chicduong** [comment]
@preetikr apologies, the changes never got pushed. please see recent changes.

File: DataConnectors/Qualys VM/profile.ps1 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Azure Functions profile.ps1
```
Comment (preetikr) [inline DataConnectors/Qualys VM/profile.ps1:1]: What's changed here? This is same as the profile.ps1 file that's part of PR#809 that we merged recently - file at  https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Qualys%20VM/profile.ps1

File: DataConnectors/Qualys VM/requirements.psd1 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This file enables modules to be automatically managed by the Functions service.
```
Comment (preetikr) [inline DataConnectors/Qualys VM/requirements.psd1:1]: What's changed here? This is same as the profile.ps1 file that's part of PR#809 that we merged recently - file at
https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Qualys%20VM/requirements.psd1

File: DataConnectors/Qualys VM/AzureFunctionQualysVM/run.ps1 [hunk 3]
```diff
@@ -201,7 +200,6 @@ if (-not ($response.HOST_LIST_VM_DETECTION_OUTPUT.RESPONSE.HOST_LIST -eq $null))
         # Convert the array containing all the records to JSON and API POST to Log Analytics Workspace                                                                 
         $json = $newcustomObjects | ConvertTo-Json -Compress -Depth 3                                                            
         Post-LogAnalyticsData -customerId $customerId -sharedKey $sharedKey -body ([System.Text.Encoding]::UTF8.GetBytes($json)) -logType $TableName
-        Write-Output $json
```
Comment (preetikr) [inline DataConnectors/Qualys VM/AzureFunctionQualysVM/run.ps1:204]: Wouldn't removing this cause the json output to not be sent to the Log Analytics data collection API? Or is this a logging statement?

File: DataConnectors/Qualys VM/profile.ps1 (hunk 1 above)
Comment (chicduong) [reply DataConnectors/Qualys VM/profile.ps1:1]: no changes here, these are just generic/default files created with every function app

File: DataConnectors/Qualys VM/requirements.psd1 (hunk 2 above)
Comment (chicduong) [reply DataConnectors/Qualys VM/requirements.psd1:1]: no changes here, these are just generic/default files created with every function app. Also a requirement for the error we recieved

File: DataConnectors/Qualys VM/AzureFunctionQualysVM/run.ps1 (hunk 3 above)
Comment (chicduong) [reply DataConnectors/Qualys VM/AzureFunctionQualysVM/run.ps1:204]: the json object is posted into log analytics, this is writing out the output in the logs /console and is unnecessary

**Comment by chicduong** [comment]
continued on https://github.com/Azure/Azure-Sentinel/pull/815

**Pull request closed without merging**